docs/spec/index.html

```diff
@@ -1,740 +1,934 @@
-        that showcases the appropriate and ethical use of technology.  Any such solution
-        should provide a net positive social benefit, and be cognizant of the variety
-        of principles that must be balanced when building and deploying a solution.
-        The W3C TAG document on
-        <a href="https://w3ctag.github.io/ethical-web-principles/">Ethical Web Principles</a>
-        provides a strong enumeration of these principles and considerations for how to
-        best implement a solution with those principles in mind.
+        The
+        <a href="https://nvlpubs.nist.gov/nistpubs/ir/2021/NIST.IR.8397.pdf">
+          Guidelines on Minimum Standards for Developer Verification of Software
+          - NISTIR 8397
+        </a>
+        MUST be followed by developers implementing solutions intended
+        to be conformant with this specification.
+
+        <br /><br />
+        NB: this guidance
+        applies to sections beyond Software Supply Chain issues, and
+        many of the topics covered have discrete sections in this
+        specification or supplemental aids such as the
+        <a
+          href="https://github.com/w3c-ccg/traceability-interop/tree/main/tests/postman">test
+          suite</a>
+        provided in the repository for this specification.
       </p>
     </section>
-  </body>
-</html>
+  </section>
+
+  <section class="informative">
+    <h2>Disaster Recovery</h2>
+    <p>
+      Industry best practices related to disaster recovery should be
+      followed by implementations compliant with this specification,
+      specifically at a minimum guidance from the following whitepapers
+      and specifications SHOULD be used:
+    </p>
+    <ul>
+      <li>
+        <a
+          href="https://csrc.nist.gov/publications/detail/white-paper/2020/04/24/protecting-data-from-ransomware-and-other-data-loss-events/final">
+          Protecting Data from Ransomware and Other Data Loss Events
+        </a>
+      </li>
+      <li>
+        <a
+          href="https://nvlpubs.nist.gov/nistpubs/cswp/NIST.CSWP.10012020-draft.pdf">
+          Securing Data Integrity Against Ransomeware Attacks
+        </a>
+      </li>
+      <li>
+        <a href="https://csrc.nist.gov/publications/detail/sp/1800-11/final">
+          NIST SP 1800-11 - Data Integrity: Recovering from Ransomware and Other
+          Destructive Events
+        </a>
+      </li>
+      <li>
+        <a
+          href="https://docs.aws.amazon.com/whitepapers/latest/disaster-recovery-workloads-on-aws/AWS - disaster-recovery-options-in-the-cloud.html">
+          AWS - Disaster recovery options in the cloud
+        </a>
+      </li>
+    </ul>
+    <p>
+      NB: the above list links out to multiple NIST and other standards,
+      many of which are referenced directly in this specification, but
+      each of which should be carefully considered and reviewed even if
+      not linked to directly from this specification.
+    </p>
```

Comment:
Notes like this should PRECEDE (not follow) the list about which they're speaking, as they are *much* more likely to be noticed there.
```suggestion
```